crawl-ref/source/mutation.cc

```diff
@@ -2514,6 +2517,8 @@ string mutation_desc(mutation_type mut, int level, bool colour,
         ostr << mdef.have[level - 1] << sanguine_armour_bonus() / 100 << ")";
         result = ostr.str();
     }
+    else if (mut == MUT_MP_WANDS && you.has_mutation(MUT_HP_CASTING))
+        result = "You expend health (3 HP) to strengthen your wands.";
```

Comment:
An alternative might be to prevent this might on dj

Reply:
I sort of like this, though? It's consistent!